app/src/main/res/layout/add_profile_activity.xml

```diff
@@ -52,7 +58,7 @@
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent"
             app:shadow_radius="8"
-            app:srcCompat="@{@drawable/ic_default_avatar}"/>
+            app:srcCompat="@{@drawable/ic_add_profile}"/>
```

Comment:
This still seems wrong @Ryggs. We shouldn't be changing this compat in this PR.

Reply:
Still unresolved @Ryggs.

Reply:
Hi @BenHenning what should I be doing here? I since reverted to its original state but looks like that is not the solution? Thanks

Reply:
You should use line that's currently in develop here:

https://github.com/oppia/oppia-android/blob/ac68f842d3f9c1816f0e0c1ff3e5e9b9ea165774/app/src/main/res/layout/add_profile_activity.xml#L55.

Reply:
Thanks @BenHenning working on this ATM

Reply:
Hi @BenHenning , the line from develop has now been used to maintain the current state, PTAL